ArtOfIllusion/src/artofillusion/ScrollViewTool.java

```diff
@@ -227,17 +212,17 @@ else if (navigationMode == ViewerCanvas.NAVIGATE_TRAVEL_LANDSCAPE)
 
 	public void mouseStoppedScrolling()
 	{
-	    if (window != null && boundCamera != null)
+        if (window != null && boundCamera != null)
         {
-            boundCamera.getCoords().copyCoords(camera.getCameraCoordinates());
-            if (boundCamera.getObject() instanceof SceneCamera) ((SceneCamera)boundCamera.getObject()).setDistToPlane(distToPlane);
+            //boundCamera.getCoords().copyCoords(camera.getCameraCoordinates());
```

Comment:
Unused code should be removed. Were you keeping it for a reason?

Reply:
Yes, it was meant to be removes. Residue from the earlier life.